### PR TITLE
Feature/scp 64 navbar brushup

### PR DESCRIPTION
**Pull request by SRenuuu**

### CODE REVIEW

File: src/components/NavbarSection/CartPopup/CartPopupItemsList.tsx [hunk 1]
```diff
@@ -3,21 +3,24 @@ import {Col, Row} from "react-bootstrap";
 import CartPopupItem from "./CartPopupItem";
 
 const CartPopupItemsList: React.FC = () => {
+    const itemData = [
```
Comment (IsuruVihan) [inline src/components/NavbarSection/CartPopup/CartPopupItemsList.tsx:6]: Better to add the type as well (We might need that in the future)

File: src/components/NavbarSection/CartPopup/CartPopupButton.tsx [hunk 2]
```diff
@@ -12,38 +12,23 @@ const CartPopupButton: React.FC<CartPopupButtonProps> = (props) => {
     const cartCount = props;
```
Comment (IsuruVihan) [inline src/components/NavbarSection/CartPopup/CartPopupButton.tsx:12]: you can take an approach like
`const { value } = props;`
So you don't need the period to access the props.

**Review by IsuruVihan — APPROVED** [review]
Everything looks fine!

File: src/components/NavbarSection/CartPopup/CartPopupButton.tsx (hunk 2 above)
Comment (SRenuuu) [reply src/components/NavbarSection/CartPopup/CartPopupButton.tsx:12]: Cool, will implement that 😃 Thanks for the tip 👍🏻

**Pull request merged**